include/openssl/ess.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef HEADER_ESS_H
+# define HEADER_ESS_H
+
+# include <openssl/opensslconf.h>
+
+# ifndef OPENSSL_NO_ESS
+# ifdef  __cplusplus
+extern "C" {
+# endif
+# include <openssl/safestack.h>
+# include <openssl/x509.h>
```

Comment:
All directives after `# ifndef OPENSSL_NO_ESS` need to be indented one more space

Reply:
Also, make sure to include `openssl/esserr.h` here.